app/page.tsx

```diff
@@ -87,6 +88,102 @@ let columnsShown = false;
 
 type NimbusExperiment = types.experiments.NimbusExperiment;
 
+/**
+ * Takes message data from `lib/asrouter-local-prod-messages/data.json` and
+ * appends any message data collected from Looker that does not already exist
+ * (ie. no duplicate message ids). The Looker message data is also cleaned up
+ * to match the message data objects from asrouter, remove any test messages,
+ * and update surface tags.
+ */
```

Comment:
It seems like it could be more maintainable (ie easier for folks with partial or missing context to reason about) to split this up into a number of functions.

Perhaps:
* one function that does the reading, querying, and writing
* a clean_data function
* a merge_data function

What do you think?

Reply:
Yeah, I agree this would be cleaner and more maintainable. I'll try to split it up like this